vsts/scripts/tagRunTimeImagesForRelease.sh

```diff
@@ -12,10 +12,23 @@ sourceBranchName=$BUILD_SOURCEBRANCHNAME
 outFilePmeMCR="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/$acrPmeProdRepo-runtime-images-mcr.txt"
 sourceFile="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/runtime-images-acr.txt"
 
+if [[ ! -f "$sourceFile" ]]; then
+			echo "Creating consolidated runtime image file '$sourceFile'..."
+			touch "$sourceFile"
```

Comment:
Might have to do `mkdir -p` the directory repo here as well.
https://github.com/microsoft/Oryx/pull/2194/files#r1362827563

Reply:
Might have to do `mkdir -p` the directory repo here as well, before the `touch`.
https://github.com/microsoft/Oryx/pull/2194/files#r1362827563